### ISSUE TITLE
Axices colors

### ISSUE THREAD

**Issue by ondister** [beto-rodriguez/LiveCharts2#201]
How could I create  the x and y axises with colors and thicknesses different from the separate lines of the x and y axises?

**Comment by langejay** [comment]
I would like to change the colour dynamically (from a setting/value in our application).  Hard setting the colour and stroke when creating the plot is not quite what we are after.

I use the following code...

XAxis = View.Chart.XAxes.First();

And can change a number of things about the axis like Name and MaxLimit.  But I cannot see how I can modify the axis display (colour and thickness).  There seems to be properties that I cannot set (in code) ... i.e. SerperatorsPaint and SeperatorsBrush

Any help appreciated.

**Comment by beto-rodriguez** [comment]
The `XAxes` property returns an `IAxis` instance, cast it to `Axis` and you will se the props

**Comment by langejay** [comment]
YES!  Thanks you!  I actually just figured it out myself.  Whilst I have your attention though, the difference between setting the separators and the actual axis itself (the bottom and left most lines) what properties should i be looking at (Black for the bottom and left and light grey for the separators by way of example)

**Comment by langejay** [comment]
YES!  Thanks you!  I actually just figured it out myself.  Whilst I have your attention though, the difference between setting the separators and the actual axis itself (the bottom and left most lines) what properties should i be looking at (Black for the bottom and left and light grey for the separators by way of example)
